src/AppBundle/Controller/AdminScoreController.php

```diff
@@ -62,9 +62,9 @@ public function resultAction($project, $username)
 
         $userTable = $lh->getTable("user", $dbName);
         $pageTable = $lh->getTable("page", $dbName);
-        $loggingTable = $lh->getTable("logging", $dbName);
-        $revisionTable = $lh->getTable("revision", $dbName);
-        $archiveTable = $lh->getTable("archive", $dbName);
+        $loggingTable = $lh->getTable("logging_userindex", $dbName);
+        $revisionTable = $lh->getTable("revision_userindex", $dbName);
+        $archiveTable = $lh->getTable("archive_userindex", $dbName);
```

Comment:
You just broke backwards compatibility with all versions of MediaWiki.  There's a reason we go through the labs helper - specifically to allow for a table mapping.  

That should be documented better.